### ISSUE TITLE
Cavitating pulmonary mass

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#7091]
**Preferred term label:**
Cavitating pulmonary mass

**Synonyms**

**Definition (free text, please give PubMed ID)**
Thick-walled abnormal gas-filled space within interstitial mass ( > 30 mm).

**Parent term (use hpo.jax.org/app)**
pulmonary cavity

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
cavitating lung malignancy
cavitating lung coccidiomycosis
cavitating lung nocardiosis
cavitating lung cryptococcosis
granulomatosis with polyangiitis
cavitating rheumatoid nodules
cavitating pulmonary infarct

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik We already added a term for this
https://hpo.jax.org/app/browse/term/HP:0033831
But I will update the definition and add this new parent.
Here is the proposed new definition based on your suggestion above.
Thick-walled abnormal gas-filled interstitial mass within a lung with a diameter greater than 30 mm.

If this definition still needs work please reopen this term request.
